charactersheet/charactersheet/templates/dm/encounter.tmpl.html

```diff
@@ -0,0 +1,95 @@
+<div class="row">
+  <div class="col-xs-12">
+    <div class="h2">
+      Encounters<br/>
+      <small>Create and manage events and sections of your campaign. Group encounters under a common topic, or...</small>
+    </div>
+    <div class="row">
+      <div class="col-xs-12 col-sm-4">
+        <div class="panel panel-default">
+          <div class="panel-body">
+            <div class="panel-heading">
+              <div class="h4">
+                Encounters
+                <div class="pull-right">
+                  <span class="glyphicon glyphicon-plus" data-bind="click: openAddModal"></span>
```

Comment:
Yeah, this was something I'd intended to do, but never got around to. I agree it's needed.